doc/x3/tutorial/minimal.qbk

```diff
@@ -105,24 +105,23 @@ generates function definitions that are meant to be placed in cpp files.
 [note `BOOST_SPIRIT_DECLARE` is variadic and may be used for one or more rules.
 Example: `BOOST_SPIRIT_DECLARE(r1, r2, r3);`]
 
-In this example, the top rule is `employee`. We declare `employee` in this
-header file:
+In this example, the top rule is `employee`. We define `employee` rule
+placeholder in this header file:
 
     namespace client
     {
         namespace parser
         {
             namespace x3 = boost::spirit::x3;
-            using employee_type = x3::rule<class employee, ast::employee>;
-            BOOST_SPIRIT_DECLARE(employee_type);
+            x3::rule<class employee_r, ast::employee> const employee = "employee";
+            BOOST_SPIRIT_DECLARE(decltype(employee));
```

Comment:
That would be the same problem as previous, was ambiguous `employee_class` will become `employee_id`

Reply:
And what's the difference with employee_r ?

Reply:
actually, what's wrong with employee_class? it's just a name.

Reply:
Ugh, read it: employee class. While it is technically a class it has no direct relation to employee class which is ast::employee.

If you want to revert the name change, go ahead and amend the commit, I will not object.

Reply:
I'm OK with *_id or *_class. *_r does not make any sense. I'd prefer it if you do the change since you are the one issuing the PR and I am the one reviewing it.

Reply:
I'm leaning towards *_id, since we are looking at a big-ish change anyway. It is succinct and easily understandable.

Reply:
I do not want to waste my time, sorry.